whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapServiceTestIntegration.java

```diff
@@ -2773,38 +2836,63 @@ private void assertTnCNotice(final Notice notice, final String value) {
         assertThat(notice.getLinks().get(0).getValue(), is(value));
     }
 
-    private void assertPersonalRedaction(final Entity entity, final int redaction, final AttributeType attribute) throws JsonProcessingException {
-        final String entityJson = getObjectMapper().writeValueAsString(entity);
+    private void assertMultipleValuesRedaction(final RdapObject rdapObject, final String prefix, final AttributeType type,final String multipleValues) {
+        final String rdapAttrName = (type == LANGUAGE) ? "lang" : type.getName();
+        final Redaction redaction = rdapObject.getRedacted().stream().filter( redaction1 -> redaction1.getPostPath()!= null && redaction1.getPostPath().contains(rdapAttrName)).findFirst().get();
```

Comment:
redactedField instead of redaction1? I can get a better name

Reply:
No it will not work as its a map and maps are not ordered so has to be like this